moxin-backend/src/backend_impls.rs

```diff
@@ -795,15 +839,17 @@ pub struct BackendImpl {
 impl BackendImpl {
     /// # Argument
     ///
+    /// * `home_dir` - The home directory of the application.
     /// * `models_dir` - The download path of the model.
     /// * `max_download_threads` - Maximum limit on simultaneous file downloads.
     pub fn build_command_sender(
+        home_dir: String,
         models_dir: String,
         max_download_threads: usize,
     ) -> Sender<Command> {
         wasmedge_sdk::plugin::PluginManager::load(None).unwrap();
 
-        let sql_conn = rusqlite::Connection::open(format!("{models_dir}/data.sql")).unwrap();
+        let sql_conn = rusqlite::Connection::open(format!("{home_dir}/data.sql")).unwrap();
```

Comment:
Separating these is a good idea 👍🏽 

Side-note: since the data.sql is the actual database file, and not a set of SQL instructions, shouldn't the extension be something like `.sqlite`? (merely to avoid confusion)

Reply:
Okay, it seems like it's using .sqlite, which is a SQLite database.